Orange/widgets/unsupervised/owpca.py

```diff
@@ -57,8 +58,9 @@ class Error(widget.OWWidget.Error):
 
     def __init__(self):
         super().__init__()
-        self.data = None
+        ConcurrentWidgetMixin.__init__(self)
```

Comment:
I'm still pretty sure this will initialize `ConcurrentWidgetMixin` twice, but if this is the more correct way of doing it I'm happy to start writing my code like this.

Reply:
It does not call `__init__` twice - I tested. :) Also, f I delete that line the widget stops working.